src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
Combine selectors with the same styles 
![image](https://github.com/mate-academy/layout_stars/assets/99403198/4d257cbb-7a13-4179-8968-bb4e01a5fa55)